modules/340-extended-monitoring/images/extended-monitoring-exporter/src/extended-monitoring.py

```diff
@@ -301,8 +304,10 @@ def do_GET(self):
             return
 
         if self.path == "/healthz":
-            self.send_response(200)
+            # Fail if metrics were last collected more than 15 minutes ago
+            self.send_response(200 if self.__class__._last_observe > (datetime.now() - timedelta(minutes=15)) else 500)
```

Comment:
For the sake of readability of the decision making

```suggestion
            fresh_enough = self.__class__._last_observe > (datetime.now() - timedelta(minutes=15))
            self.send_response(200 if fresh_enough else 500)
```